Engine/Module/ModuleDebugDraw.cpp

```diff
@@ -396,7 +396,134 @@ bool ModuleDebugDraw::Init()
     APP_LOG_SUCCESS("Module Debug Draw initialized correctly.")
 
 	return true;
-}	
+}
+
+//void ModuleDebugDraw::Render()
```

Comment:
Why is this huge amount of code commented?

Reply:
Some leftover code from the last merge